domains/mertal-i.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "mertal-i",
+           "email": "",
+           "discord": "756092748104335370",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.SKwpppmoVeC4eo8UVv9DF7inwpr6T1c7CJ1VULOurIk7f97GfyJtC4CjQBlZgv6ozPXsIzWejztvds2d0jDIoJ2__3D5cN58iztxzso4a_AukvAyaDo3CiqOa_VVU9clnRpE78F1oixzDCdzGkYTxShqf0ufHact_JZeJtQe6Nlvo7u25MULuXDXWrRB-8sydkAF7bFlpDt9RoN5TQMQT0chhFniiRW6GvNcv7wA0ww9XGvM3TnMUJDiEW-33P9lseFEZ7dnL00ymGQiv_NZ5udJ52d5zSBFtjyUS1neAWlAmSg2IAdI1vSOrJioXQ6uKm8g6U3V2AugXgq_DQZigA.cWNDprOdJItlfuXKLATXsw._ok99P9vK5jnDZ4wT7_R0kRjWGKhlEyWC3798tYoKv6gq1lEvcxfk52TdGu4EkDBtCHZNcJfzvQa4l2-ghvLxUEKkyHNz9O0s_Mxm1you-fAOMUr-1VhB73VIKKuYmrB.ctQR3Ro5bLhtvS4D4t4F6Q"
+        },
+    
+        "record": {
+            "A": ["185.27.134.223"]
```

Comment:
I cannot reach your website
185.27.134.223 didn’t send any data.

Reply:
my real website: this[https://mertali.rf.gd](url)

Reply:
when ı entered dns records in the A record typing the 185.27.134.223 because

Reply:
@mertal-i Your site redirects to a link that doesn't exist, are you sure you have a website? If you don't there's no rush in trying to getting this domain before someone else since its way unlikely for someone to steal the one you want.

Reply:
![image](https://github.com/user-attachments/assets/7c5639ad-a02b-4b14-ad06-17ea3e953c67)
this screen need to be pop-up. And it's my website.